libs/remix-ui/app/src/lib/remix-app/components/modals/enter.tsx

```diff
@@ -0,0 +1,60 @@
+import React, {useContext, useEffect, useState} from 'react'
+import {AppContext} from '../../context/context'
+import {UsageTypes} from '../../types'
+import { type } from 'os'
+
+interface EnterDialogProps {
+  hide: boolean,
+  handleUserChoice: (userChoice: UsageTypes) => void,
+}
+
+const EnterDialog = (props: EnterDialogProps) => {
+  const [visibility, setVisibility] = useState<boolean>(false)
+  const {showEnter} = useContext(AppContext)
+
+  useEffect(() => {
+    setVisibility(!props.hide)
+  }, [props.hide])
+
+  const enterAs = async (uType) => {
+    props.handleUserChoice(uType)
+  }
+
+  const modalClass = (visibility && showEnter) ? "d-flex" : "d-none"
+  return (
+    <div
+      data-id={`EnterModalDialogContainer-react`}
+      data-backdrop="static"
+      data-keyboard="false"
+      className={"modal  " + modalClass}
+      role="dialog"
+    >
+      <div className="modal-dialog align-self-center pb-4" role="document">
+        <div
+          tabIndex={-1}
+          className={'modal-content remixModalContent mb-4'}
+          onKeyDown={({keyCode}) => {
+          }}
+        >
+          <div className="modal-header d-flex flex-column">
+            <h3 className='text-dark'>Welcome to Remix IDE</h3>
+            <div className='d-flex flex-row pt-2'>
+              <h6 className="modal-title" data-id={`EnterModalDialogModalTitle-react`}>
+                To load the project with the most efficient setup we would like to know your experience type.
```

Comment:
If it's how a user uses Remix, then the dialog should not ask "your experience type." It's not good practice to ask a person what "type" they are. It creates friction in the user's mind because they need to think about converting their task into a "type." It's much better to ask "How do you plan to use Remix?" Or, "How do you primarily use Remix?" And, then the answers should be "production of assets" or even more specific. Or, something like "deploying contracts" or "testing code" or "quick prototyping" or "testing/debugging" etc. etc. And indeed, these answers need to be discussed and vetted as to our goals for capture, and then properly mapped to our other usage stats. (See my first comment above in outdated area for other suggestions).

Reply:
Let's discuss this on the call. I feel like we all have slightly different ideas about why we need this :)
I suggest merging the PR and creating the second one for updating the texts.